src/host_capabilities/crypto.rs

```diff
@@ -23,8 +23,16 @@ pub enum CertificateEncoding {
 }
 
 /// Verify_cert verifies cert against the passed cert_chain
-pub fn verify_cert(cert: Certificate, cert_chain: Option<Vec<Certificate>>) -> Result<bool> {
-    let req = CertificateVerificationRequest { cert, cert_chain };
+pub fn verify_cert(
```

Comment:
Please add some inline documentation about:

* `cert_chain`: what happens when it's `None`
* `not_after`: RFC 3339 time format string, what happens when this is `None`

Basically copy the inline docs of the struct you've added inside of `mod.rs`

Reply:
Done in https://github.com/kubewarden/policy-sdk-rust/pull/71/commits/197ec3110e48226165bb7c623f3055b10ef8854b